src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -139,6 +142,34 @@ public String getDataInventory(String params) {
         return data;
     }
 
+    public String getReloadToken(InputStream input) {
+        try {
+            BufferedReader in = new BufferedReader(new InputStreamReader(input));
+            String currentLine;
+            while ((currentLine = in.readLine()) != null) {
+                if (currentLine.contains("&reloadToken=")) {
+                    String token;
```

Comment:
You can get rid of this variable, just move closing to a finally block and inline the return